### DISCUSSION TITLE
Possible to disable "installing pip dependencies" each time I run?

### DISCUSSION THREAD

**Discussion by Maple38** [hlky/stable-diffusion-webui#848] [Q&A]
I mean, shouldn't they be installed after the first run of webui.cmd? Why do I have to sit through this long step each time I want to use SD?

**Comment by codedealer** [comment]
Yep, cmd files should be separated into `install`, `update` and `run` ones.

**Reply by Strathelme1** [reply]
you can comment out the two lines like so 
![image](https://user-images.githubusercontent.com/110069505/189221629-b610b5cb-6254-487f-9529-d13694c1e399.png)

**Reply by codedealer** [reply]
You can of course, it will just be easier for people to choose when to update and when to just run the old build they got by choosing from two different cmd files.
